server/src/test/java/org/apache/uniffle/server/ShuffleServerConfTest.java

```diff
@@ -20,22 +20,16 @@
 import java.io.File;
 
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import uk.org.webcompere.systemstubs.environment.EnvironmentVariables;
-import uk.org.webcompere.systemstubs.jupiter.SystemStub;
-import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
 
 import org.apache.uniffle.common.util.ByteUnit;
+import org.apache.uniffle.storage.EnvTestBase;
```

Comment:
It looks weird to extend a `storage` test from `server` package.

Reply:
New test writers may not be aware of this `EnvTestBase`, because it's hiding in the `storage` package.
And I think this kind of abstraction isn't saving us much work.